bip-0352.mediawiki

```diff
@@ -284,7 +283,6 @@ The receiver obtains the public key from the ''scriptSig''. The receiver MUST pa
 
 The sender and receiver MUST calculate an input hash for the transaction in the following manner:
 
-* Let ''A = A<sub>1</sub> + A<sub>2</sub> + ... + A<sub>n</sub>'', where each ''A<sub>i</sub>'' is the public key of an input from the ''[[#inputs-for-shared-secret-derivation|Inputs For Shared Secret Derivation]]'' list<ref name="why_include_A"></ref>
```

Comment:
One thing about removing this line is that now when reading the spec from top to bottom this section references `A` without any context. Only after reading the below sections does the spec tell the reader to return here with the value for `A`. Perhaps we can move this section below sending and receiving and in those sections tell the reader to see below, or we can add a note here saying `A` is specified below depending on being sender or receiver.

Reply:
Good catch, @andrewtoth . Reading through this again, I think it makes more sense to remove the Input Hash section entirely and just define it inline. I opened https://github.com/bitcoin/bips/pull/1629 for this